libyara/include/yara/types.h

```diff
@@ -186,6 +186,15 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
     ((x) != NULL ? (x)->type == EXTERNAL_VARIABLE_TYPE_NULL : true)
 
 
+// The types of states in AC automaton
+#define YR_ATOM_TYPE_ANY                     0x00
+#define YR_ATOM_TYPE_CLASS                   0xCC
+#define YR_ATOM_TYPE_MASKED_LITERAL_0F       0x0F
```

Comment:
It looks like `YR_ATOM_TYPE_MASKED_LITERAL_0F` and `YR_ATOM_TYPE_MASKED_LITERAL_F0` are not used.